### ISSUE TITLE
Consider removing the renovate validator from local pre-commit linting

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse#4566]
## Problem

Once installed, the renovate validator is fast. However, it's a gnarly dependency to install and takes _forever_ compared to the other modules (literally comparing minutes with renovate validator to seconds with the others, even on my overpowered computer!)

## Description

Maybe we can rely just on CI for this one?

## Alternatives

Keep it and accept that it takes a long time to create the pre-commit envs.

If there is a binary distribution of the tool, we could include it in `ov` base image, and use it directly, instead of the pre-commit hook which installs it from scratch with node-gyp.

## Additional context

It's mostly come up for me as a result of working on `ov`, and `ov clean` forcing a recreation of the pre-commit environments. Normally it isn't an issue, but I could see it as a frustrating part of environment setup.

**Comment by dhruvkb** [comment]
I would agree to remove the validator from the pre-commit and pre-push stages. 

We can accomplish this by making that particular hook `manual` and then only running it in CI using the specific flag `--hook-stage` flag. From the docs:

> The manual stage (via stages: [manual]) is a special stage which will not be automatically triggered by any git hook -- this is useful if you want to add a tool which is not automatically run, but is run on demand using pre-commit run --hook-stage manual [hookid].

There is a recent discussion to make a dedicated validator image https://github.com/renovatebot/renovate/discussions/29804, hopefully something useful to us comes out of that discussion.
